deploy-satellite-update-playbook.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
@sthirugn I would argue that doesn't seem like ansible convention... but I think being verbose is fine in this case, since users who aren't familiar with ansible may be using this tool. Keeping "playbook" is fine with me.